script/constructron.lua

```diff
@@ -1298,13 +1258,19 @@ me.get_job = function()
                                 find_path = true
                             end
 
+                            local landfill_check

```

Comment:
if you provide a default here you can get rid of the `or false` in the portion below
```suggestion
                            local landfill_check = false
```

Reply:
done